lore/settings.py

```diff
@@ -139,3 +141,26 @@ def get_var(name, default):
 STATICFILES_DIRS = (
     os.path.join(BASE_DIR, 'lore', 'static'),
 )
+
+# e-mail configurable admins
+ADMIN_EMAIL = get_var('LORE_ADMIN_EMAIL', None)
+if ADMIN_EMAIL is not None:
+    ADMINS = (('Admins', ADMIN_EMAIL),)
+
+CAS_ENABLED = False
+
+# Optional CAS Integration
+if get_var('LORE_USE_CAS', None):
+    CAS_ENABLED = True
+    MIDDLEWARE_CLASSES += ('django_cas_ng.middleware.CASMiddleware',)
+    INSTALLED_APPS += ('django_cas_ng',)
+    AUTHENTICATION_BACKENDS += ('django_cas_ng.backends.CASBackend',)
+    LOGIN_URL = '/login/'
+    LOGIN_REDIRECT_URL = '/'
+    CAS_SERVER_URL = get_var(
+        'LORE_CAS_URL', 'https://auth-r.mitx.mit.edu/login'
+    )
+    CAS_EXTRA_LOGIN_PARAMS = {
+        'provider': 'touchstone',
+        'appname': "LORE"
+    }
```

Comment:
They should always be static for LORE per the spec, but we could make them configurable if/when we need to.